src/ripple/test/jtx/impl/Env_test.cpp

```diff
@@ -414,6 +414,22 @@ class Env_test : public beast::unit_test::suite
     }
 
     void
+    testMemo()
+    {
+        using namespace jtx;
+        Env env(*this);
+        env.fund(XRP(10000), "alice");
+        env(noop("alice"), memodata("data"));
+        env(noop("alice"), memoformat("format"));
+        env(noop("alice"), memotype("type"));
+        env(noop("alice"), memondata("format", "type"));
+        env(noop("alice"), memonformat("data", "type"));
+        env(noop("alice"), memontype("data", "format"));
+        env(noop("alice"), memo("data", "format", "type"));
+        env(noop("alice"), memo("data1", "format1", "type1"), memo("data2", "format2", "type2"));
```

Comment:
The purpose of `Env_test` is to test the jtx framework, not to test memos. This just makes sure that all the funclets compile correctly. A full blown memo test is certainly something we could use - thanks for volunteering to write it!